vidarr-server/src/main/resources/db/migration/V0007__analysisext_idx.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX IF EXISTS external_external_id_analyis_id;
+CREATE INDEX external_external_id_analyis_id ON analysis_external_id(analysis_id);
```

Comment:
Should this be named `analysis_external_id_analysis_id`?

Reply:
It looks like the same content got pushed, and it's still `external_external_id_analysis_id`

Reply:
Spelling is hard.